gammapy/maps/wcs.py

```diff
@@ -634,6 +635,30 @@ def solid_angle(self):
 
         return u.Quantity(dx * dy, 'sr')
 
+    def get_region_idx(self, region):
+        """Return idx of pixels inside region
+
+        TODO: implement list of region for each axis
+
+        Parameters
+        ----------
+        region : `~regions.PixelRegion` or `~regions.SkyRegion` object
+            A region on the sky could be defined in pixel or sky coordinates.
+        """
+
+        from regions import PixCoord
+        # TODO : if Pixel Compound regions are taken into account, rather convert to PixelRegion
```

Comment:
Remove this TODO?
I don't understand it, and it doesn't seem to be needed?

Reply:
Compound regions will be needed for exclusion masks, no?